app/contact/page.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Page = () => {
```

Comment:
Maybe you can update the component to be more informative about what it's purpose is? Consider using Contact as the name of the Page.